cmd/ebrelayer/relayer/ethereum.go

```diff
@@ -392,9 +392,6 @@ func (sub EthereumSub) logToEvent(networkDescriptor oracletypes.NetworkDescripto
 	if decodedEvent, err := bridgeBank.BridgeBankFilterer.ParseLogBurn(cLog); err == nil {
 		event.ClaimType = ethbridgetypes.ClaimType_CLAIM_TYPE_BURN
 		event.To = append(event.To, decodedEvent.To...)
-		// burn doesn't have symbol or name
-		//event.Symbol = decodedEvent.Symbol
-		//event.Name = decodedEvent.Name
```

Comment:
This will be addressed in a separate issue https://app.zenhub.com/workspaces/current-sprint---engineering-615a2e9fe2abd5001befc7f9/issues/sifchain/sifnode/2553